packages/fxa-auth-server/test/scripts/move-customers-to-new-plan.ts

```diff
@@ -365,7 +365,8 @@ describe('CustomerPlanMover', () => {
     it("returns false if the customer does not have a price that's excluded", () => {
       const result = customerPlanMover.isCustomerExcluded([
         {
-          ...subscription1,
+          // TODO: Either provide full mock, or reduce type required isCustomerExcluded
```

Comment:
nit: should we file a bug for this so we track/bury it in the backlog?

Reply:
We do this as unknown as Type pattern quite a bit. Maybe it's fine. I'll let @StaberindeZA make the call.